doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17369,16 +17360,13 @@ WaterHeater:HeatPump,
     ,                        !- Inlet Air Temperature Schedule Name
     ,                        !- Inlet Air Humidity Schedule Name
     ,                        !- Inlet Air Zone Name
-    WaterHeater:Mixed,       !- Tank Object Type
     HPWHPlantTank,           !- Tank Name
```

Comment:
@nmerket The object type fields should remain.  The pair of fields for type and name ensure that the reference to another object will be unique.  I will comment further in the idd.